release-1.8/release_notes_draft.md

```diff
@@ -76,6 +95,19 @@ support for self hosting the cluster control plane.
 
 [SIG Cluster Lifecycle]: https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle
 
+### SIG Instrumentation
+
+[SIG Instrumentation][] is responsible for metrics production and
+collection.
+
+For the 1.8 release, SIG Instrumentation focused on stabilizing the APIs
+and components that are required to support the new version of the Horizontal Pod
+Autoscaler API: the resource metrics API, custom metrics API, and
+metrics-server, which is the new replacement for Heapster in the default monitoring
```

Comment:
`metrics server` vs `metric-server`.  Should we mention Metric Server as given at https://github.com/kubernetes-incubator/metrics-server, or use  `metric-server` ?